src/main/kotlin/org/kamiblue/client/module/modules/combat/AutoArmor.kt

```diff
@@ -22,12 +22,17 @@ internal object AutoArmor : Module(
     modulePriority = 500
 ) {
     private val delay = setting("Delay", 5, 1..10, 1)
-
     private val timer = TickTimer(TimeUnit.TICKS)
     private var lastTask = TaskState(true)
 
+    var isPaused = false
+
     init {
+        onDisable {
+            isPaused = false
+        }
         safeListener<TickEvent.ClientTickEvent> {
+            if (isPaused) return@safeListener
```

Comment:
isPaused should be reset after 10 seconds or something if AutoMend isn't active, to prevent issues where isPaused possibly isn't reset 

Currently I only see it reset when AutoMend is disabled... which is a problem if someone disabled AutoMend's `Disable On Complete` setting

Reply:
I changed pause conditions